install/kubernetes/helm/istio/charts/sidecar-injector/webhook-create-signed-cert.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
+set -e
+
+usage() {
+    cat <<EOF
+Generate certificate suitable for use with an Istio webhook service.
```

Comment:
oh I think @costinm mentioned to me today this isn't necessary and we should just use the Istio CA

Reply:
Does it mean `sidecar-injector` should make `istio-ca` as its dependency?

Reply:
Wait... you should be able to do auto side car without istio security enabled, right?   cc @ayj

Reply:
CA should be a mandatory component of Istio even for non-auth environment. It will generate secrets but the proxy does not necessarily use those secrets for encryption.